crates/ruff_python_formatter/tests/snapshots/black_compatibility@cases__line_ranges_diff_edge_case.py.snap

```diff
@@ -25,16 +25,14 @@ print ( "format me" )
 ```diff
 --- Black
 +++ Ruff
-@@ -6,7 +6,8 @@
- # This can be fixed in the future if we use a better diffing algorithm, or make Black
- # perform formatting in a single pass.
- 
--print ( "format me" )
+@@ -9,5 +9,5 @@
+ print ( "format me" )
```

Comment:
This Black test documents a limitation of Black. Ruff does not have this limitation